### PR TITLE
Feature/account page styling

### PR DESCRIPTION
**Pull request by MariamM110**
- changed header to be in the middle
- added a selection of avatars to change in user profile
- some minor changes in styling across different files

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -11,12 +11,60 @@ import "./App.css";
...
+  const baseUrl = "https://chrome-dino-game.s3.amazonaws.com/assets/";
+
   const changeUser = (username) => {
     setCurrentUser(username);
   };
 
-  const profile =
-    "https://chrome-dino-game.s3.amazonaws.com/assets/dino-idle.png";
+  const changeProfileAvatar = (avatar) => {
+    console.log(`${baseUrl}${itemData[avatar - 1]["img"]}`);
```
Comment (louisglan) [inline src/App.js]: this console.log needs removing

File: src/App.js [hunk 2]
```diff
@@ -11,12 +11,60 @@ import "./App.css";
...
+    setCurrentAvatar(`${baseUrl}${itemData[avatar - 1]["img"]}`);
```
Comment (louisglan) [inline src/App.js:56]: I think using 'avatar - 1' renders the id useless - might be worth removing the id keys from each object in itemData

**Comment by yuna-iwata** [comment]
netlify deploy has failed

File: src/Components/AccountPageSubPages/AvatarSelection.jsx [hunk 3]
```diff
@@ -1,3 +1,87 @@
...
+  ImageListItem,
+  Tooltip,
+  IconButton,
+} from "@mui/material";
+import { changeAvatar } from "../../Networking";
+
+export default function AvatarSelection({ username, changeProfileAvatar }) {
+  const avatarBaseUrl = "https://chrome-dino-game.s3.amazonaws.com/assets/";
+
+  const itemData = [
```
Comment (louisglan) [inline src/Components/AccountPageSubPages/AvatarSelection.jsx]: Could have this passed down as a prop from App. Same with baseUrl

File: src/Components/AccountPageSubPages/AvatarSelection.jsx [hunk 4]
```diff
@@ -1,3 +1,87 @@
...
+        bgcolor: "background.paper",
+        boxShadow: 1,
+        borderRadius: 2,
+        p: 2,
+        minWidth: 300,
+      }}
+    >
+      <Typography sx={{ p: 2 }}>Select an avatar</Typography>
+      <ImageList sx={{ width: 500, height: 200 }} cols={3} rowHeight={1.5}>
+        {itemData.map((item) => (
```
Comment (louisglan) [inline src/Components/AccountPageSubPages/AvatarSelection.jsx]: I feel this should map onto a component which should have it's own file

**Review by louisglan — CHANGES_REQUESTED** [review]
In addition to below changes, should be made to deploy to netlify

**Review by louisglan — APPROVED** [review]
Would be nice to change game.key !== null to game.key but otherwise everything looks great

**Pull request merged**